src/Libraries/Microsoft.Extensions.AI/ChatCompletion/ChatClientStructuredOutputExtensions.cs

```diff
@@ -24,7 +24,8 @@ public static class ChatClientStructuredOutputExtensions
     {
         IncludeSchemaKeyword = true,
         DisallowAdditionalProperties = true,
-        IncludeTypeInEnumSchemas = true
+        IncludeTypeInEnumSchemas = true,
+        RequireAllProperties = true,
```

Comment:
With AIFunction, someone who wants to turn on strict can do so by adding "Strict"=true to AdditionalProperties.

Do we have a way to do that for response format? Maybe we need to add an AdditionalProperties to ForJsonSchema / ChatResponseFormat?

Reply:
I think we need to expose an AIJsonSchemaCreateOptions parameter to these methods so that the default can be overridden

Reply:
I've reverted this change and exposed an additional parameter in the `ForJsonSchema` factory.